lib/tasks/link-audit.rake

```diff
@@ -10,28 +10,37 @@ task link_audit: :environment do
 
   def fetch(url_string)
     url = URI(url_string)
-    response = Net::HTTP.get_response(url)
```

Comment:
This PR is much easier to read if you ignore whitespace: https://github.com/gabebw/hotline-webring/pull/177/files?w=1